app/models/holiday.rb

```diff
@@ -21,7 +21,7 @@ class Holiday < ActiveRecord::Base
   belongs_to :user
 
   validates_presence_of :user, :start, :end
-  validates_date :start, on_or_after: :today
+  validates_date :start, on_or_after: Date.today
```

Comment:
:today kommt wohl vom validates_timeliness Gem und das benutzt (anscheinend ?) eine andere Zeitzone, als Date.today. Deshalb haben die Tests zwischen 0 und 1 Uhr gefailt ...
In der Beschreibung des Gems steht zwar, dass es die Zeitzone von ActiveRecord übernimmt, aber das scheint irgendwie nicht zu funktionieren.